expression/builtin_arithmetic_vec.go

```diff
@@ -515,11 +515,11 @@ func (b *builtinArithmeticModRealSig) vecEvalReal(input *chunk.Chunk, result *ch
 	if err := b.args[0].VecEvalReal(b.ctx, input, result); err != nil {
 		return err
 	}
+	result.MergeNulls(buf)
 	x := result.Float64s()
 	y := buf.Float64s()
 	for i := 0; i < n; i++ {
 		if buf.IsNull(i) {
```

Comment:
Try to use `result.IsNull(i)` here(ut failed on not generating same warnings as non-vec version did),  turned out that following code need to deal with situation where `result.isNull(i)` is true.